agent.go

```diff
@@ -423,6 +480,16 @@ func main() {
 		},
 	}
 
+	if len(os.Args) == 1 && filepath.Base(os.Args[0]) == "init" && os.Getpid() == 1 {
```

Comment:
I would prefer if we don't force a specific name here. I think `os.Getpid() == 1` should be enough to know if this agent has been run as the init process.